README.md

```diff
@@ -54,6 +54,29 @@ export default MyComponent extends Component {
 }
 ```
 
+### Reads, then Writes
+
+Since the purpose of this addon is to minimize layout thrashing, we want to perform _all_ *reads* before we perform *writes*. Additionally, we want to encourage closing over values created during the *read* phase that are used in the *write* phase. This helps ensure that work doesn't leak outside of the frame we're performing the work in. 
```

Comment:
"encourage" is too soft, if you read intermixed with writing you will cause layout thrashing, defeating the whole point. Writing should always derive state from state derived during read.  It doesn't have to be a closure it could be stored any way, but it can't come from the dom during write.

Reply:
Updated to be stronger wording.